src/lib.rs

```diff
@@ -115,6 +115,8 @@ extern crate smithay_client_toolkit as sctk;
 extern crate calloop;
 
 pub mod dpi;
+#[macro_use]
```

Comment:
Do we really want to expose the macro? From the looks of it, the macro won't even compile if users use it.

Reply:
`#[macro_use]` doesn't publicly export macros - that's `#[macro_export]`. That just lets the macro be used elsewhere in the codebase.